app/models/incident.rb

```diff
@@ -102,9 +104,55 @@ def csv_row
     row
   end
 
+  def export_claims_xml!
+    report = driver_incident_report
+    builder = Nokogiri::XML::Builder.new do |doc|
+      doc.claims_incident_data do
+        doc.incident do
+          doc.date_entered        Date.today.strftime('%Y-%m-%d')
```

Comment:
```suggestion
          doc.date_entered        created_at.strftime('%Y-%m-%d')
```
?

Reply:
That was my thought originally too, but the specification is that this should be the date that it is entered into the external system (i.e. whenever we're sending the record along).